test/functional/ui/statuscolumn_spec.lua

```diff
@@ -912,8 +962,8 @@ describe('statuscolumn', function()
     command('call setline(1, range(1, 99))')
```

Comment:
Maybe this test should use `v:lnum` instead of `%l` now?

Reply:
Irrelevant I think but sure.